customfields/custom_fields.go

```diff
@@ -0,0 +1,60 @@
+package customfields
+
+import (
+	"encoding/xml"
+	"io"
+	"strings"
+)
+
+// CustomFields is a string to string map of custom field names to values.
+// Ref: https://articles.braintreepayments.com/control-panel/custom-fields
+type CustomFields map[string]string
+
+type xmlField struct {
+	XMLName xml.Name
+	Value   string `xml:",chardata"`
+}
+
+var nameMarshalReplacer = strings.NewReplacer("_", "-")
```

Comment:
The other SDKs do this explicitly. It might be a side effect of the other SDKs converting en-mass all underscore variables to hyphenated variables before XML serialization. It seems the control panel doesn't allow hyphens in custom fields and I guess that prevents overlap.